plugins/net_plugin/net_plugin.cpp

```diff
@@ -2049,9 +2049,9 @@ namespace eosio {
 
       auto reset_on_failure = [&]() REQUIRES(sync_mtx) {
          sync_source.reset();
-         sync_known_lib_num = chain_info.lib_num;
+         sync_known_froot_num = chain_info.fork_root_num;
```

Comment:
`froot` here and all other `fork_root` locations.

Reply:
I think I would rather go the other direction. Change `froot` to `fork_root` except in logging.

Reply:
whatever we choose let's be consistent throughout in the code.

Reply:
I prefer `fork_root` as it spells out what it really is; it is not long. The `f` in `froot` can be anything, it can even be be thought of `finality`.

Reply:
Renamed `froot` to `fork_root`.